package.json

```diff
@@ -8,7 +8,7 @@
     "url": "git://github.com/Khan/KaTeX.git"
   },
   "files": [
-    "katex.js",
+    "dist/katex.js",
```

Comment:
In fact, I think this line edit should be reverted.  It's probably good to include the root (ES6) `katex.js` in the NPM package (why not? then people can choose). `dist` is already fully included 3 lines below.  It's the `main` key, 6 lines above, that seems to need to change.